README.md

```diff
@@ -1,14 +1,33 @@
-# Compute@Edge default template for Rust
+# Compute@Edge default starter kit for Rust
 
-Default package template used by the [Fastly CLI][cli] for Rust-based Compute@Edge projects.
+Get to know the Fastly Compute@Edge environment with a basic starter that demonstrates routing, simple synthetic responses, and overriding caching rules.
 
-## Usage
-- Install the [Fastly CLI][latest] to your `$PATH`
-- Run `fastly compute init`
+**For more details about this and other starter kits for Compute@Edge, see the [Fastly developer hub](https://developer.fastly.com/solutions/starters)**
 
-[cli]: https://github.com/fastly/cli
-[latest]: https://github.com/fastly/cli/releases/latest
+## Features
 
-## Security issues
+* Add headers to an inbound request
+* Allow only requests with particular HTTP methods
+* Match request URL path and methods for routing
+* Build synthetic responses at the edge
+* Send requests to a backend
+* Override caching policy response headers
 
-Please see our [SECURITY.md](SECURITY.md) for guidance on reporting security-related issues.
```

Comment:
The link to our `SECURITY.md` in the readme is a requirement from our security team for all open-source projects. Please add back in at the bottom of the page.